app/src/main/java/com/codingblocks/cbonlineapp/notifications/NotificationsActivity.kt

```diff
@@ -107,10 +111,15 @@ class NotificationsActivity : BaseCBActivity() {
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         return when (item.itemId) {
             R.id.action_clear -> {
-//                if (notificationDao.allNotificationNonLive.isNotEmpty()) {
-//                    showconfirmation()
-//                }
-                true
+                runBlocking {
+                    val a =withContext(Dispatchers.Default){
```

Comment:
why have you used Dispatchers.Default ?

Reply:
To make sure I get the boolean response first from DB queries. I thought to make it async await hence I used w`withContext(Dispatchers.Default)` to achieve using coroutines.

Reply:
Why default dispatcher?